web-api/src/ingestion/routes.rs

```diff
@@ -73,7 +75,8 @@ pub(super) fn configure_service(config: &mut ServiceConfig) {
                 .route(web::get().to(get_document_property))
                 .route(web::put().to(put_document_property))
                 .route(web::delete().to(delete_document_property)),
-        );
+        )
+        .service(web::resource("/_silo_management_api").route(web::post().to(silo_management_api)));
```

Comment:
:art:/:wrench: all the routes are directed at apis, so we could/should drop the suffix in the route and function here as well.
```suggestion
        .service(web::resource("/_silo_management").route(web::post().to(silo_management)));
```